### PR TITLE
Switch to SUSE Enterprise Storage 2.1

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: crowbar_framework/config/repos-cloud.yml [hunk 1]
```diff
@@ -137,6 +117,26 @@ suse-12.1:
...
+      name: "SUSE-Enterprise-Storage-2.1-Pool"
+      required: "optional"
+      features: ["ceph"]
+      repomd:
+        tag: "obsproduct://build.suse.de/SUSE:SLE-12-SP1:Update:Products:SES2.1/ses/2.1/POOL/x86_64"
+        key_md5: "9a62177e1c6852d48453ed3909956d6b"
+      url:
+      smt_path: "SUSE/products/storage/2.1/x86_64/product"
+      ask_on_error: false
+    suse-enterprise-storage-2.1-updates:
```
Comment (tboerger) [inline crowbar_framework/config/repos-cloud.yml]: Does it make sense to add it yet anyway? There is no repository for that.

Comment (tboerger) [inline crowbar_framework/config/repos-cloud.yml]: Ah forget it, there are other repos with the same comment.

**Comment by tboerger** [comment]
:+1:

**Comment by mjura** [comment]
This depends on https://github.com/crowbar/crowbar/pull/2185

File: crowbar_framework/config/repos-cloud.yml [hunk 2]
```diff
@@ -39,26 +39,6 @@ suse-12.0:
...
-    suse-enterprise-storage-2-updates:
-      name: "SUSE-Enterprise-Storage-2-Updates"
-      required: "optional"
-      features: ["ceph"]
-      repomd:
-        tag: "obsrepository://build.suse.de/SUSE:Updates:Storage:2:x86_64/update"
-        key_md5: "9a62177e1c6852d48453ed3909956d6b"
-      url:
-      smt_path: "SUSE/Updates/Storage/2/x86_64/update"
-      ask_on_error: false
```
Comment (vuntz) [inline crowbar_framework/config/repos-cloud.yml]: Can you simply drop the whole section about SLE12?

File: crowbar_framework/config/repos-cloud.yml [hunk 3]
```diff
@@ -137,6 +117,26 @@ suse-12.1:
       url:
       smt_path: "SUSE/Updates/SLE-HA/12-SP1/x86_64/update"
       ask_on_error: false
+    suse-enterprise-storage-2.1-pool:
+      name: "SUSE-Enterprise-Storage-2.1-Pool"
+      required: "optional"
+      features: ["ceph"]
+      repomd:
+        tag: "obsproduct://build.suse.de/SUSE:SLE-12-SP1:Update:Products:SES2.1/ses/2.1/POOL/x86_64"
```
Comment (vuntz) [inline crowbar_framework/config/repos-cloud.yml]: Is the check based on tag working right now?

File: crowbar_framework/config/repos-cloud.yml [hunk 4]
```diff
@@ -137,6 +117,26 @@ suse-12.1:
...
+        key_md5: "9a62177e1c6852d48453ed3909956d6b"
+      url:
+      smt_path: "SUSE/products/storage/2.1/x86_64/product"
+      ask_on_error: false
+    suse-enterprise-storage-2.1-updates:
+      name: "SUSE-Enterprise-Storage-2.1-Updates"
+      required: "optional"
+      features: ["ceph"]
+#      repomd:
+#        tag: "obsrepository://build.suse.de/SUSE:Updates:Storage:2:x86_64/update"
```
Comment (vuntz) [inline crowbar_framework/config/repos-cloud.yml]: s/2/2.1/

File: crowbar_framework/config/repos-cloud.yml (hunk 2 above)
Comment (mjura) [inline crowbar_framework/config/repos-cloud.yml]: It will be done in another commit https://github.com/crowbar/crowbar-core/commit/ba5fd6523a7ec1fcb8b8bf135f1124ec2412198c

File: crowbar_framework/config/repos-cloud.yml (hunk 3 above)
Comment (mjura) [inline crowbar_framework/config/repos-cloud.yml]: For me this repository works fine, what check should work?

File: crowbar_framework/config/repos-cloud.yml (hunk 4 above)
Comment (mjura) [inline crowbar_framework/config/repos-cloud.yml]: Done

File: crowbar_framework/config/repos-cloud.yml (hunk 3 above)
Comment (vuntz) [inline crowbar_framework/config/repos-cloud.yml]: If you go to the repositories page in the crowbar webui, it checks that the repo is valid. If it's all green, then you're good.

Comment (mjura) [inline crowbar_framework/config/repos-cloud.yml]: sles12sp1repos

Comment (mjura) [inline crowbar_framework/config/repos-cloud.yml]: Yeap it is green

**Comment by vuntz** [comment]
+1

File: crowbar_framework/config/repos-cloud.yml [hunk 5]
```diff
@@ -137,6 +72,26 @@ suse-12.1:
...
       ask_on_error: false
+    suse-enterprise-storage-2.1-pool:
+      name: "SUSE-Enterprise-Storage-2.1-Pool"
+      required: "optional"
+      features: ["ceph"]
+      repomd:
+        tag: "obsproduct://build.suse.de/SUSE:SLE-12-SP1:Update:Products:SES2.1/ses/2.1/POOL/x86_64"
+        key_md5: "9a62177e1c6852d48453ed3909956d6b"
+      url:
+      smt_path: "SUSE/products/storage/2.1/x86_64/product"
```
Comment (vuntz) [inline crowbar_framework/config/repos-cloud.yml]: Actually, this is: SUSE/Products/Storage/2.1/x86_64/product

Comment (mjura) [inline crowbar_framework/config/repos-cloud.yml]: Thank you, done

**Comment by mjura** [comment]
I've added also changes to `repos-ses.yml` file

**Comment by tboerger** [comment]
Still :+1:

**Comment by rsalevsky** [comment]
:+1:
